src/Domain/InterfaceHelpers.hpp

```diff
@@ -268,12 +268,21 @@ SPECTRE_ALWAYS_INLINE constexpr auto interface_apply(
           std::forward<ExtraArgs>(extra_args)...);
 }
 
+// The `box` argument to this function overload is not constrained to
+// `db::DataBox` to work around an issue with GCC <= 8.
+//
+// Details on the issue:
+//
+// GCC <= 8 tries to instantiate the `db::DataBox<DbTagsList>` template even
+// when this function overload is _not_ SFINAE-selected. In that case the
```

Comment:
can you use if constexpr instead of SFINAE?

Reply:
I think no because the overloads take different arguments and different template parameters. I can't think of a way to use `constexpr` instead, but I can easily be mistaken.

Reply:
I think no because the overloads take different arguments and different template parameters. I can't think of a way to use `constexpr` instead, but I can easily be mistaken.